src/components/WhaleSightingViewer/WhaleSightingViewer.scss

```diff
@@ -0,0 +1,54 @@
+@import "../../App.scss";
+
+* {
+    font: $fontType;
+}
+
+h2 {
+   font-size: $titleSizeDesktop;
+}
+
+.whale-sighting-page {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 30% 90%;
+}
+
+.whale-sighting-heading{
+    grid-row-start: 1;
+    grid-row-end: 2;
+    grid-column-start: 1;
+    grid-column-end: 2;
+}
+
+.whale-sighting-map-view-button{
+    grid-row-start: 1;
+    grid-row-end: 2;
+    grid-column-start: 1;
+    grid-column-end: 2;
+}
+
+.whale-sighting-filters{
+    grid-row-start: 1;
+    grid-row-end: 2;
+    grid-column-start: 2;
+    grid-column-end: 3;
+}
+
+.whale-sighting-sort{
+    grid-row-start: 1;
+    grid-row-end: 2;
+    grid-column-start: 3;
+    grid-column-end: 4;
+}
+
+.whale-sighting-post{  
+    grid-row-start: 2;
+    grid-row-end: 3;
+    grid-column-start: 1;
+    grid-column-end: 4;
+
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr;
+}
```

Comment:
Use flex here instead of grid probably!

Here's the nice flex guide: https://css-tricks.com/snippets/css/a-guide-to-flexbox/

I would recommend:
```scss
.whale-sighting-posts {
    display: flex;
    justify-content: space-between;
    flex-wrap: wrap;

    .whale-sighting-post {
        width: 30%;
    }
}
```